content/news/053/index.md

```diff
@@ -121,6 +121,33 @@ _Check out the [web examples][chuot-examples]!_
 
 ## Library Updates
 
+### Godot Rapier: Rewrite it in Rust
+
+{{ video_figure(
+    type="video/mp4",
+    alt="Godot + Rapier logo make up the 'Godot Rapier' logo ; a few balls fall on it, showing off soft physics.",
+    src="godot_rapier.mp4",
+    caption="Godot Rapier showcase") }}
+
+[Godot Rapier][godot_rapier_github] is a Physics library for [Godot][godot], using [Rapier][rapier]:
```

Comment:
[Godot Rapier][godot_rapier_github] written by [@Ughuuu](https://github.com/Ughuuu) is a ..